src/test/java/steps/Hook.java

```diff
@@ -0,0 +1,31 @@
+package steps;
+
+import base.BaseUtil;
+import cucumber.api.Scenario;
+import cucumber.api.java.After;
+import cucumber.api.java.Before;
+import io.github.bonigarcia.wdm.ChromeDriverManager;
+import org.openqa.selenium.chrome.ChromeDriver;
+
+public class Hook extends BaseUtil {
+
+    private BaseUtil base;
+
+    public Hook(BaseUtil base) {
+        this.base = base;
+    }
+
+    @Before
+    public void initializeTest() {
+        ChromeDriverManager.getInstance().setup();
+        base.driver = new ChromeDriver();
+    }
+
+    @After
+    public void TearDownTest(Scenario scenario) {
+        if (scenario.isFailed()) {
+            System.out.println(scenario.getName());
```

Comment:
Instead of using the stdio, you cold use some logging framework like log4j or slf4j. Which are more sophisticated and standard for taking care of these activates.

Reply:
I think It is not necessary to add logger here because the assert always logging the failure message. 
I think take a screenshot is more useful here.